### PR TITLE
Misc test fixes

### PR DESCRIPTION
**Pull request by Droogans**
JIRA: N/A 

### LGTM's
- [x] Dev LGTM
- [x] Zoom LGTM

### CODE REVIEW

File: test/package.json [hunk 1]
```diff
@@ -35,7 +33,7 @@
     "webdriver:update": "scripts/webdriver-update"
   },
   "ava": {
-    "snapshotLocation": "test/snapshots",
+    "snapshotDir": "snapshots",
```
Comment (CITguy) [inline test/package.json]: should this be `snapshots` (as defined here) or `screenshots` (as defined in the .gitignore above)?

Comment (Droogans) [reply test/package.json]: I'll write this up in the wiki tomorrow and clarify what each suite is and does. A DOM snapshot is not the same as a visreg screenshot.

Comment (Droogans) [reply test/package.json]: I think a way to make it "unwikiable" is to make it more obvious.

I'm going to change the layout to something like this:

```
test/
  artifacts/
    visregScreenshots/
    domSnapshots/
```

I will include a README.md file in each directory, explaining what each one is used for.

Comment (Droogans) [reply test/package.json]: Checking this into version control with README.md files is an alright idea, but they get us back to a state where docs require pull requests to update, which is bad for obvious reasons.

I'll make sure README.md files remain short, and point to the wiki for the bulk of the information around the purposes of these directories.

**Comment by Droogans** [comment]
https://github.com/rackerlabs/helix-ui/wiki/Testing#dom-snapshots added.

File: test/visreg/visreg.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ export function suite(browserName: string) {
     test.before(async () => {
         const config: IConfig = {
             browser: browserName,
-            screenshotsDir: "visreg/screenshots",
+            screenshotsDir: "artifacts/visregScreenshots",
```
Comment (Droogans) [inline test/visreg/visreg.ts]: I wonder....should this be `artifacts/screenshots/visreg`?

I'm fine as-is, I think it looks good....I could do either.

Comment (CITguy) [reply test/visreg/visreg.ts]: I'm indifferent.

**Pull request merged**